common/include/pcl/make_shared.h

```diff
@@ -36,7 +36,7 @@
 
 #pragma once
 
-#warning "Do not use this header, its contents have been moved to pcl/memory.h"
+PCL_DEPRECATED_HEADER(1, 12, "Use <pcl/memory.h> instead.")
```

Comment:
Warning needs to exist for 2 cycles. This was there in 1.10, so it will be removed in 1.13, right?

Reply:
As you say.

Reply:
Same for `point_traits.h` then? It was deprecated after 1.10.